docs/csharp/asynchronous-programming/snippets/generate-consume-asynchronous-streams/finished/Program.cs

```diff
@@ -139,9 +139,16 @@ await foreach (var issue in RunPagedQueryAsync(client, PagedIssueQuery, "docs")
         // Stop with 10 pages, because these are large repos:
         while (hasMorePages && (pagesReturned++ < 10))
         {
-            var postBody = issueAndPRQuery.ToJsonText();
+            var postBody = JsonConvert.SerializeObject(issueAndPRQuery);
+
+            var requestHeaders = new HttpHeaders
+        {
+            {"Content-Type", "application/json"},
+            {"Accept", "application/json"}
+        };
+
```

Comment:
This code will never work. [`HttpHeaders` is an abstract class](https://learn.microsoft.com/dotnet/api/system.net.http.headers.httpheaders), and if you intend to [instantiate `HttpRequestHeaders` there's no `.ctor`](https://learn.microsoft.com/dotnet/api/system.net.http.headers.httprequestheaders). I suggest, you instead leave the call as it was, and reformat the new lines as desired.

```suggestion
```